client/src/hooks/useMyDataset.js

```diff
@@ -16,6 +16,36 @@ export const useMyDataset = () => {
   } = useContext(MyDatasetContext)
   const { token, email, userFormat, setUserFormat } = useScPCAPortal()
 
+  const emptyDatasetProjectOptions = {
+    includeBulk: false,
+    includeMerge: false,
+    modalities: []
+  }
+
+  const [defaultProjectOptions, setDefaultProjectOptions] = useState(
+    emptyDatasetProjectOptions
+  )
+
+  // Prepopulate user's project download options in the modal
```

Comment:
```suggestion
  // When myDataset changes, update the default options for adding additional projects.
```

If we change this to no longer be in a modal, this comment would confuse more than help. Specifically say what the important piece about this code is or anything that is non-obvious. If you think that we need a comment at all.